homeworks/lesia.turua_lesiaturuta/PokemonProject/scripts/pokemon.js

```diff
@@ -327,28 +106,66 @@ const filterPokemons = (checkBig, checkSmall) => {
     return newPokemons;
 };
 
-const init = () => {
-// disabled for html elements
-    selectSearchEl.disabled = true;
-    selectSearchRadioOnlyNewEl.disabled = true;
-    selectSearchRadioFavoritesEl.disabled = true;
+const filterPokemonsByType = (type) => pokemons.filter((pokemon) => pokemon.type.includes(type));
+
+const init = async () => {
+    // get pokemons
```

Comment:
Don't you think, `init` function is too big?
You event add a comment to describe what is going on in this piece of code. Proly you can move it to the function with this very name `getPokemons`